gapir/client/session.go

```diff
@@ -309,6 +309,14 @@ func (s *session) newADB(ctx context.Context, d adb.Device, abi *device.ABI) err
 	if err != nil {
 		return log.Errf(ctx, err, "Getting gapid.apk files directory")
 	}
+	appDir, err := apk.AppDir(ctx)
+	if err != nil {
+		return log.Errf(ctx, err, "Getting gapid.apk directory")
+	}
+
+	// Ignore the error return from this. This is best-effort.
```

Comment:
`returned from this`. Please also link to the ndk hack in the comment.

Reply:
Done